client/pages/App.jsx

```diff
@@ -40,7 +41,8 @@ const router = createBrowserRouter(
             }
         >
             <Route path='/create' element={<CreatePostPage />}/>
-            <Route path='/feed' element={/* Eventually, this should be a verified page */ <PostFeedPage />} />
+            <Route path='/feed' element={<PostFeedPage />} />
```

Comment:
token is required, so eventually this will be a verified page; please return the comment